pom.xml

```diff
@@ -33,10 +35,16 @@
     <dependencies>
         <dependency>
             <groupId>org.spigotmc</groupId>
-            <artifactId>spigot</artifactId>
+            <artifactId>spigot-api</artifactId>
             <version>1.16.1-R0.1-SNAPSHOT</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-core</artifactId>
+            <version>2.13.3</version>
+            <scope>provided</scope>
+        </dependency>
```

Comment:
Why is that added?

Reply:
I don't know if you noticed, but the last 1600+ merged PR contained code that referenced classes provided by internal spigot dependencies.

This is why they changed the dependency from `spigot-api` to `spigot`, rendering the entire project unworkable unless you had the server jar built in your local repo.

I reverted this, and added a provided dependency for what those classes referenced.